components/ErrorNotification/index.js

```diff
@@ -0,0 +1,13 @@
+import { notification } from "antd";
+
+export function notifyError(error) {
+  //This print is intentional and not a debug print
+  console.log(error);
+
+  notification.error({
```

Comment:
This function is not correct. It should show the user the error in order for them to know why it was not possible to show them why the action was not possible. And because the type of errors will depend on the API call, I think it will be very difficult to generalize all the errors. 

This just works as a backfall notification error when you don't know what happened.

Reply:
Yes, I think it is a good way to do it now :+1: